build/Dockerfile.ubuntu

```diff
@@ -9,12 +9,15 @@ RUN echo 'deb http://ppa.launchpad.net/brightbox/ruby-ng/ubuntu trusty main' >>
     rm -rf /var/lib/apt/lists/* && \
     gem install bundler
 
-ADD server/Gemfile /build/server/Gemfile
-ADD server/Gemfile.lock /build/server/Gemfile.lock
+COPY server/Gemfile /build/server/Gemfile
+COPY server/Gemfile.lock /build/server/Gemfile.lock
```

Comment:
BTW in general, it might be slightly better to use `COPY server/Gemfile server/Gemfile.lock /build/server/` to reduce the number of image layers?